### ISSUE TITLE
lohView fails with error when samples have data in differing chromosomes

### ISSUE THREAD

**Issue by zlskidmore** [griffithlab/GenVisR#239]
lohView seems to fail when data is in 1 sample for a chromosome but not in the other

For example:
```r
tmpA <- HCC1395_Germline[HCC1395_Germline$chrom == "1",]
tmpA$sample <- "test"
tmpB <- HCC1395_Germline[HCC1395_Germline$chrom == "2",]
tmpC <- rbind(tmpA, tmpB)
lohView(x=tmpC)
```

Results in:
```r
Did not detect the prefix chr in the chromosome column of x... adding prefix
genome specified is preloaded, retrieving data...
 Show Traceback
 
 Rerun with Debug
 Error in 1:num : NA/NaN argument In addition: Warning messages:
1: In min(as.numeric(as.character(x$position))) :
  no non-missing arguments to min; returning Inf
2: In FUN(X[[i]], ...) : NAs introduced by coercion to integer range
3: In max(as.numeric(as.character(x$position))) :
  no non-missing arguments to max; returning -Inf
4: In FUN(X[[i]], ...) : NAs introduced by coercion to integer range
```
@jkunisak, can you look into this when you have a chance?

**Comment by jkunisak** [comment]
Hey Zach,Yep I will try to have a look at it before Thursday's meeting. Just to be clear though, you want to plot LOH differences for the sample "test" on only chromosome 1 and the sample HCC1395 for only chromosome 2?  Jason KunisakiBiology Major with Concentration in Genomics and Computational Biology
College of Arts & SciencesWashington University in St. Louis | Class of 2017jhk275@yahoo.com | 310-625-2195 

    On Tuesday, February 16, 2016 4:28 PM, Zachary Skidmore <notifications@github.com> wrote:
 

 lohView seems to fail when data is in 1 sample for a chromosome but not in the otherFor example:tmpA <- HCC1395_Germline[HCC1395_Germline$chrom == "1",]
tmpA$sample <- "test"
tmpB <- HCC1395_Germline[HCC1395_Germline$chrom == "2",]
tmpC <- rbind(tmpA, tmpB)
lohView(x=tmpC)Results in:Did not detect the prefix chr in the chromosome column of x... adding prefix
genome specified is preloaded, retrieving data...
 Show Traceback

 Rerun with Debug
 Error in 1:num : NA/NaN argument In addition: Warning messages:
1: In min(as.numeric(as.character(x$position))) :
  no non-missing arguments to min; returning Inf
2: In FUN(X[[i]], ...) : NAs introduced by coercion to integer range
3: In max(as.numeric(as.character(x$position))) :
  no non-missing arguments to max; returning -Inf
4: In FUN(X[[i]], ...) : NAs introduced by coercion to integer range@jkunisak, can you look into this when you have a chance?—

**Comment by zlskidmore** [comment]
Yeah, that was just an example to reproduce the error. For a more realistic example I would imagine that the same error would occur if you omitted the allosomes for male samples but not for female, which could happen if the user did that their self in the input data instead of using the M/F vector option.

I would imagine it's a simple fix but if it's not let me know and we can collaborate.

**Comment by zlskidmore** [comment]
@jkunisak didn't you resolve this? Maybe I just forgot to close it

**Comment by jkunisak** [comment]
Hey Zach,
Yea this should be fixed. I will double check tonight though. 

Best,
Jason Kunisaki
